crates/cheatcodes/spec/src/vm.rs

```diff
@@ -688,6 +688,10 @@ interface Vm {
     #[cheatcode(group = Evm, safety = Safe)]
     function lastCallGas() external view returns (Gas memory gas);
 
+    /// takes a signed transaction as bytes and executes it
+    #[cheatcode(group = Evm, safety = Safe)]
+    function broadcastRawTransaction(bytes calldata data) external;
```

Comment:
can we add some more docs here?

the docs mention execute but it's named broadcast, so what does this do exactly?

Reply:
Done - should be more clear now. Moved it to the broadcast section too as it was out of place